README.md

```diff
@@ -86,7 +86,35 @@ This POST creates a returns an `instanceId` which is a new resource, which popul
 1. Run `npm run build`. The build result files are under the build directory. (Note: Change the relative path of api_demo_icon.svg, if it is not deployed to root folder.)
 1. Alternatively, run `npm start` to debug the application.
 
-The below document is generated automatically by Create React App:
+## Configure the Sample
+
+The config.js file is used to configure the sample for your environment. Set values for the following variables to configure the Intelligence Server, project, folder, user credentials, and number of documents to display on the page.
```

Comment:
- rephrased the sentence
- added link to `config.js`
- removed unused user credentials since it is unused in the implementation.

```suggestion
The [config.js file](./client/src/config.js) is used to configure the sample for your environment. You can modify the values of the following variables listed in the table to configure what to display on the page, including the Intelligence Server, project, folder, and the number of documents.
```